core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.component.ts

```diff
@@ -87,6 +87,7 @@ export class TopicsAndSkillsDashboardPageComponent {
   displayedTopicSummaries: CreatorTopicSummary[] = [];
   displayedSkillSummaries: SkillSummary[] = [];
   skillStatusOptions: string[] = [];
+  prevWidth: number = window.innerWidth;
```

Comment:
This can be expanded.

Reply:
Have you tested manually resizing the browser from desktop -> mobile and back to see if that still works?

Reply:
@kevintab95 i have tested this in browser and it is working fine.

Reply:
Could you please add a testing video for the scenario described above? Thanks!

Reply:
```suggestion
  windowWidth: number = window.innerWidth;
```